configs/tasks/obj_nav_mp3d.yaml

```diff
@@ -0,0 +1,47 @@
+ENVIRONMENT:
+  MAX_EPISODE_STEPS: 750
+SIMULATOR:
+  TURN_ANGLE: 30
+  TILT_ANGLE: 30
```

Comment:
We are planning to add look_up and look_down actions to the action space for the challenge?  If so, we should add a max tilt angle parameter.

Reply:
> We are planning to add look_up and look_down actions to the action space for the challenge?

Yes, so the agent can see objects hanging on the wall or placed on the floor.

Reply:
Ah, any suggestion for the max title angle then?

Reply:
-90..+90 degrees?